File: interface/src/main/java/eu/nyerel/panda/monitoringresult/MonitoringResultService.java
package eu.nyerel.panda.monitoringresult;

import eu.nyerel.panda.monitoringresult.calltree.CallTreeNode;

import java.rmi.Remote;
import java.rmi.RemoteException;
import java.util.List;

/**
 * @author Rastislav Papp (rastislav.papp@gmail.com)
 */
public interface MonitoringResultService {

    List<CallTreeNode> getCallTree();
    void clear();

}

File: interface/src/main/kotlin/eu/nyerel/panda/monitoringresult/MonitoringResultService.kt
package eu.nyerel.panda.monitoringresult

import eu.nyerel.panda.monitoringresult.calltree.CallTreeNode

/**
 * @author Rastislav Papp (rastislav.papp@gmail.com)
 */
interface MonitoringResultService {

    fun getCallTree(): List<CallTreeNode>
    fun clear()

}
